src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+     if(findByEmail(email) == null) {
+         return false;
+     } else if (findByEmail(email).equals(email) || findByEmail(email).getPassword().equals(password)) {
+         return true;
+
+     }
+     return false;
+
```

Comment:
you should create an object of UserService class
then call method findByEmail on this object to find user

then compare passwords